ui/raidboss/data/04-sb/raid/o4s.js

```diff
@@ -1273,6 +1275,7 @@ export default {
       'missingTranslations': true,
       'replaceSync': {
         'Exdeath': 'エクスデス',
```

Comment:
```suggestion
        '(?<! )Exdeath': 'エクスデス',
```

他の言語もこのように変更してください、それならテストがパスできると思います。
もしよければ、韓国語のreplaceSyncにもネオエクスデスの翻訳（네오 엑스데스）を追加してほしいです。

Reply:
Thanks! I was thinking what should I do...